packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -248,10 +248,11 @@ export default class BuySellStore extends BaseStore {
                             this.setSearchResults([]);
                         }
                     }
-                } else if (response.error.code === 'PermissionDenied') {
+                    // Added a check to prevent console errors
+                } else if (response && response.error.code === 'PermissionDenied') {
```

Comment:
Why do we need to check for response?

Reply:
To prevent console errors that arise if response is unavailable, I'll add a check to verify if the object is present, then evaluate the value of its properties.